src/Management/src/Endpoint/SpringBootAdminClient/SpringBootAdminRefreshRunner.cs

```diff
@@ -175,7 +183,7 @@ private async Task SafeUnregisterAsync(SpringBootAdminClientOptions clientOption
         {
             try
             {
-                _logger.LogInformation("Unregistering from Spring Boot Admin Server at {Url}.", clientOptions.Url);
+                _logger.LogDebug("Unregistering from Spring Boot Admin Server at {Url}.", clientOptions.Url);
```

Comment:
```suggestion
                _logger.LogInformation("Unregistering from Spring Boot Admin Server at {Url}.", clientOptions.Url);
```
This one should only happen once, at shutdown right?